### PR TITLE
Feature/no whitespace punctuation words

### PR DESCRIPTION
**Pull request by hothanhtuan1209**
In this pull request, i refactor no_whitespace_punctuation_words.py file in practice-2, update function main and add module no_whitespace_punctuation_words to main.py
#44

### CODE REVIEW

File: python/practices/practice-3/src/path2/no_whitespace_punctuation_words.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+"""
+
+
+import string
+import os
+from helpers.file_reader import read_file
+from constants.constants import TEST
+
+
+def clean_text_and_split(test):
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path2/no_whitespace_punctuation_words.py]: Please update function's name.

**Comment by cuongdoan-agilityio** [comment]
Please check carefully before creating PR.

**Comment by hothanhtuan1209** [comment]
image

**Comment by hothanhtuan1209** [comment]
image

File: python/practices/practice-3/src/path2/no_whitespace_punctuation_words.py (hunk 1 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path2/no_whitespace_punctuation_words.py]: Done

File: python/practices/practice-3/src/path2/no_whitespace_punctuation_words.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+punctuation from the words, and converts them to lowercase.
+"""
+
+
+import os
+from helpers.file_reader import read_file
+from constants.constants import TEST
+
+
+def clean_and_split_text(test):
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path2/no_whitespace_punctuation_words.py]: Please update the function and parameter name.

File: python/practices/practice-3/src/path2/no_whitespace_punctuation_words.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+def main():
+    """
+    Reads a text file, cleans it, and prints the cleaned words.
+    """
+
+    print(os.path.basename(__file__))
+    file_content = read_file(TEST)
+
+    if file_content:
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path2/no_whitespace_punctuation_words.py]: Can we move this condition into the function `clean_and_split_text`?

File: python/practices/practice-3/src/path2/no_whitespace_punctuation_words.py (hunk 2 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path2/no_whitespace_punctuation_words.py]: Done

File: python/practices/practice-3/src/path2/no_whitespace_punctuation_words.py (hunk 3 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path2/no_whitespace_punctuation_words.py]: Done

**Pull request merged**